### ISSUE TITLE
Execute Load

### ISSUE THREAD

**Issue by patrickzurek** [cedelis/xcoaitoolkit#191]
JIRA issue created by: rcook
Originally opened: 2010-10-05 05:15 PM

Issue body:
 (nt)

**Comment by patrickzurek** [comment]
JIRA Comment by user: Chris Delis (cedelis)
JIRA Timestamp: 2010-10-13 10:07 PM

Comment body:

Conversion:

INFO  - [PRG] converted: 2860137, invalid: 1 records. It took 00:04:56.698
INFO  - [LIB] Conversion completed at 2010/10/12 16:18:41
INFO  - [LIB] Conversion statistics summary: converted: 16788403, invalid: 4 records. It took 00:53:14.538

Modify / Load:

INFO  - [LIB] Import statistics summary: created 16778135, updated: 0, skipped: 0, invalid: 2394, deleted: 0, bib: 5575308, auth: 4568549, holdings: 6634278 records. Invalid files: 0. It took 03:45:20.212. checkTime: 00:14:31.481. insertTime: 00:11:22.997. others: 03:19:25.734

Optimization of lucene index (taken from programmer.log):
010-10-12 23:55:28,122 [main](Importer.java:810) INFO  - [PRG] optimizing database...
2010-10-13 00:38:57,061 [main](Importer.java:815) INFO  - [PRG] Import done
2010-10-13 00:38:57,091 [main](Importer.java:148) INFO  - [PRG] Import finished

TOTAL INDEX TIME:
0:53:14 + 3:45:20 = 4:38:34

LUCENE OPTIMIZE:
24:38:57 - 23:55:28 = 43:29

TOTAL LOAD TIME: 5:22:03

Note: I encountered the following SAX parser exception in one of the modified xml files:

2010-10-12 17:32:01,745 [main](Importer.java:448) INFO  - [PRG] Modifying records... filename:uiu_bib.3_400000.xml
........... (10%)
......... (20%)
.[Fatal Error] :175140:31: Character reference "&#0" is an invalid XML character.
org.marc4j.MarcException: Unable to parse input
        at org.marc4j.MarcXmlParser.parse(MarcXmlParser.java:95)
        at org.marc4j.MarcXmlParser.parse(MarcXmlParser.java:64)
        at org.marc4j.MarcXmlParserThread.run(MarcXmlParserThread.java:115)
Caused by: org.xml.sax.SAXParseException: Character reference "&#0" is an invalid XML character.
        at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1231)
        at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:522)
        at org.marc4j.MarcXmlParser.parse(MarcXmlParser.java:93)
        ... 2 more
2010-10-12 17:32:03,338 [main](Importer.java:535) ERROR - [PRG] [MarcException] Unable to parse input uiu_bib.3_400000.xml lastRecordToModify: UIUdb4262144
2010-10-12 17:32:03,339 [main](Importer.java:545) INFO  - [PRG] Modify statistics for uiu_bib.3_400000.xml: converted: 2126, invalid: 0 records.  Invalid files:1.  It took 00:00:01.595

Once the import completed, I fixed the file then re-ran the importer.

Here are the statistics reported from lucene_dbStatistics.sh:

$ sh ./lucene_dbStatistics.sh 
: not foundStatistics.sh: 10: 
Lucene Statistics Value is:true
init(luceneDir): lucene_index
init(logDir): log
 log4j property file: /xc/OAIToolkit-0.6.6alpha/OAIToolkit.log4j.properties
2010-10-13 17:23:54,289 [main](Logging.java:50) INFO  - Logging started in directory: log
2010-10-13 17:23:56,297 [main](Importer.java:953) INFO  -  **************\* Lucene Database Statistics **************\* 

2010-10-13 17:23:56,298 [main](Importer.java:955) INFO  - Total records in the Lucene Database are: 16786009
2010-10-13 17:23:56,298 [main](Importer.java:956) INFO  - 
     Bibliographic records: 5583182
2010-10-13 17:23:56,298 [main](Importer.java:957) INFO  -   Authority records: 4568549
2010-10-13 17:23:56,299 [main](Importer.java:958) INFO  -   Holdings records: 6634278
2010-10-13 17:23:56,299 [main](Importer.java:959) INFO  -   Classification records: 0
2010-10-13 17:23:56,299 [main](Importer.java:960) INFO  -   Community information records: 0
2010-10-13 17:23:56,300 [main](Importer.java:961) INFO  -   Deleted records: 1
